community/ufw/build

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh -e
+
+python ./setup.py install --root="$1"
```

Comment:
` python setup.py install --root="$1"`

Reply:
Also add (before this line):

```sh
sed -i 's|/lib|/usr/lib|;s|sbin|bin|' setup.py
```

Reply:
I'm sorry. First time. Why is the `sed` replacement required?

Reply:
It changes the install location for `ufw` to `/usr/lib` instead of `/lib` and `bin` instead of `sbin`. Not 100% required and feel free to omit it if all works well regardless. :+1: